themes/default/content/blog/pulumi-release-notes-74/index.md

```diff
@@ -80,11 +82,23 @@ We released v0.5.1 of Pulumi YAML which included bug fixes, new functions, diagn
 - Allow Fn::Join to take expressions as inputs, previously the second argument had to be a syntactical list.
   [#241](https://github.com/pulumi/pulumi-yaml/pull/241)
 
+- Fix `pulumi convert` panicking on programs containing `Fn::ToJSON`,`Fn::Secret` and `Fn::Invoke` (with an empty arguments property). [#250](https://github.com/pulumi/pulumi-yaml/pull/250), [#260](https://github.com/pulumi/pulumi-yaml/pull/260), and [#262](https://github.com/pulumi/pulumi-yaml/pull/262) respectively.
+
 As always, please feel free to submit feature requests and bug reports to the [Pulumi YAML GitHub Repo](https://github.com/pulumi/pulumi-yaml). We love hearing feedback from users!
 
+### Add `PreviewDigest` for third party tools
```

Comment:
This is largely just an internal API thing - it's not part of the product - we can leave it off.